### DISCUSSION TITLE
Using fast_executemany with FreeTDS driver

### DISCUSSION THREAD

**Discussion by g-plus-theworld** [mkleehammer/pyodbc#1327] [General]
Hi,

I am facing an issue where I need to use pyodbc in a docker container and it needs to auth using Active Directory/Kerberos into an MS SQL server. I have tested that the FreeTDS driver can auth into this server with a UID and PWD string, which is brilliant because configuring Kerberos/AD on a docker container proved to be insanely difficult.

Now, I checked the wiki for fast_executemany and it writes:
This feature is "off" by default, and is currently only recommended for applications that use Microsoft's ODBC Driver for SQL Server.

I was wondering if anyone has any experience using it with FreeTDS driver?. Any issues have come up? Is it reliable? Maybe the pyodbc devs can explain why it is not recommended?

Thanks in advance!

Regards,
Gabriel

**Comment by gordthompson** [comment]
https://github.com/mkleehammer/pyodbc/wiki/fast_executemany-support-for-various-ODBC-drivers

FreeTDS ODBC "crashes CPython interpreter"

(unless it has been updated recently to support parameter arrays)
